sklearn/utils/extmath.py

```diff
@@ -244,7 +244,7 @@ def _compute_orthonormal_lobpcg(M, Q, n_iter, tol, explicit_normal_matrix):
     """Computes an orthonormal matrix using LOBPCG."""
     # Determine the normal matrix
     if explicit_normal_matrix:
-        A = safe_sparse_dot(M, M.T.conj())
```

Comment:
@glemaitre - removing .conj() here breaks the code functionality for complex-valued data matrix M
Please consider instead adding a unit test for complex M.

Reply:
We don't accept complex values in scikit-learn so we will not need it. I understand the value in the general case but we only keep the code for our use case.

Reply:
I have .conj() in quite a few spots - you may need a good global search to find them all.